src/test/java/api/requests/RequestsAPICreationTests.java

```diff
@@ -1673,26 +1674,6 @@ public void canCreatePagedRequestWithNullProxyUser() {
     assertThat(finalStatus, is(ItemStatus.PAGED.getValue()));
   }
 
-  @Test
-  public void requestCreationDoesNotFailWhenCirculationRulesReferenceInvalidNoticePolicyId() {
```

Comment:
It is still possible for the rules to refer to a non-existent notice policy when a request is created. Why has this test been removed?

Reply:
This test is not relevant because it creates a wrong rule with "some-invalid-policy" policy. Now there is no way to create a rule with non existent policy.

Reply:
I think it is still possible, only the steps to create the situation are different.

What happens if a policy is created, specified in the rules and then deleted prior to request creation?

Reply:
OK, I returned it back and little bit updated

Reply:
Thanks